mcc/roosa/cmd/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+
+	"github.com/poka-yoke/spaceflight/mcc/ttl"
+)
+
+// Node type represents the reference between data.
+type Node struct {
+	parent   *Node
+	children []*Node
+	content  *route53.ResourceRecordSet
+}
+
+func (n *Node) isRoot() bool {
+	return n.parent == nil
+}
+
+var zoneName string
+
+// Init sets the flag parsing and input validations
+func Init() {
+	flag.StringVar(&zoneName, "zonename", "", "Hosted Zone's name to traverse")
+
+	flag.Parse()
+
+	if zoneName == "" {
+		log.Fatal("Insufficient input parameters!")
+	}
+}
+
+// PrintNode prints a Node, and all its children.
+func PrintNode(node *Node, indent int) {
+	indents := ""
+	for i := 0; i <= indent-1; i++ {
+		indents += "\t"
+	}
+	extra := ""
+	for i, record := range node.content.ResourceRecords {
```

Comment:
Can't you just do:
`for i, record := range node.content.ResourceRecords[:-1] {`
That would avoid the if altogether.